qt/aqt/forms/browser.ui

```diff
@@ -583,6 +585,14 @@
     <string notr="true">Ctrl+Shift+E</string>
    </property>
   </action>
+  <action name="actionFilterToDeck">
+   <property name="text">
+    <string>BROWSING_FILTER_TO_DECK</string>
+   </property>
+   <property name="shortcut">
+    <string notr="true">Ctrl+Q</string>
```

Comment:
This is a special keystroke on Macs, so will need to be changed to something else.